src/site/layout/css.rs

```diff
@@ -46,81 +41,24 @@ pub fn place_css(dist_dir: &str, release_tag: &str) -> Result<()> {
     // Even if you're running a development build, we still respect the custom CSS version preference
     // by falling back to fetching said version from GitHub.
     if cfg!(debug_assertions) && release_tag == ORANDA_CSS_TAG {
```

Comment:
```suggestion
    if release_tag == ORANDA_CSS_TAG {
```

(forgot to remove this)